micrometer-jmx/src/main/scala/com/avast/sst/micrometer/jmx/MicrometerJmxModule.scala

```diff
@@ -13,8 +13,9 @@ import java.time.Duration
 object MicrometerJmxModule {
 
   /** Makes configured [[io.micrometer.jmx.JmxMeterRegistry]]. */
-  def make[F[_]: Sync](
+  def make[F[_]: Sync: ContextShift](
       config: MicrometerJmxConfig,
+      blocker: Blocker,
```

Comment:
Is it really necessary even for this module?

Reply:
Since adding the `Blocker` is a backward-incompatible change, I tried to prevent doing it in the future. I think the problem might happen with this module too, even though I have no proof.

Reply:
Removed 😉